### PR TITLE
EPUB & HTML command with individual functions v2

### PR DESCRIPTION
**Pull request by yashha**
Worked on #76

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,14 +187,12 @@ Puppeteer can print some basic information about the page in the PDF. The follow
...
-<template class='header-template'>
+<template class="header-template">
 	My header
 </template>
 
-<template class='footer-template'>
-	<div class='text center'>
-		<span class='pageNumber'></span>
-	</div>
+<template class="footer-template">
```
Comment (yashha) [inline README.md]: Can't revert this :(

**Comment by danburzo** [comment]
Thank you for working on this! I was away for the holidays, I'll review the code soon. (Thanks for the patience)

**Comment by NightMachinary** [comment]
@danburzo Please merge these in.

**Comment by danburzo** [comment]
I've pulled everything over to the `epub` branch, to make it compatible with the latest changes on master.

**Pull request merged**